node/Dockerfile

```diff
@@ -19,12 +19,11 @@ RUN --mount=type=cache,target=/root/.cache --mount=type=cache,target=/go \
 
 ADD . .
 
-# RUN apt-get update
-# RUN apt-get install libcap2-bin -y
+RUN if [[ "$network" != "devnet" ]]; then apt-get update && apt-get install libcap2-bin -y; fi
 
 RUN --mount=type=cache,target=/root/.cache --mount=type=cache,target=/go \
   go build -race -gcflags="all=-N -l" --ldflags '-extldflags "-Wl,--allow-multiple-definition"' -mod=readonly -o /guardiand github.com/alephium/wormhole-fork/node
 
-# RUN setcap cap_ipc_lock=+ep /guardiand
+RUN if [[ "$network" != "devnet" ]]; then setcap cap_ipc_lock=+ep /guardiand; fi
```

Comment:
Nice catch, we need this in testnet & mainnet, perhaps doesn't work for devnet.

Reply:
Yes, it doesn't work on github CI.